app/poros/restaurant.rb

```diff
@@ -23,10 +24,11 @@ def initialize(restaurant)
     @longitude = restaurant[:coordinates][:longitude].to_s
     @location = restaurant[:location]
     @categories = restaurant[:categories]
+    @url = restaurant[:url]
```

Comment:
Added website after it was shaken so users can get all the information they want from the restaurant's website directly